tests/p2p/bootstrap_test/config.yaml

```diff
@@ -0,0 +1,36 @@
+namespace: ''
+
+config_map_name: conf
+
+script_on_exit: '' # Uncomment this to save logs './k8s/log-client-pods.sh'
+
+genesis_delta: 150
+single_pod_ready_time_out: 60
+deployment_ready_time_out: 120 #default 120
+config_path: '../config.toml'
+
+bootstrap:
+  image: 'spacemeshos/go-spacemesh-dev:go_auto'
```

Comment:
change back to develop as it won't update? especially if you're not loading the image from the config

Reply:
fixed in 207fd37bd289dcc691ce4b4207a102bc0f009af7